pkg/index/reader.go

```diff
@@ -0,0 +1,47 @@
+package index
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+)
+
+const NotAvailable = "N/A"
+
+type Reader struct {
+	f *os.File
+	csv.Reader
+}
+
+func Open(path string) (*Reader, error) {
```

Comment:
This is unused function. Do you plan to use it?
if no - we can remove `f *os.File` and `Clsoe()` function.

Reply:
Yes, I plan to import it from maven-index-crawler since they should share the TSV reader.

- maven-index-crawler: Write TSV files
- trivy-java-db: Read TSV files

Reply:
Added a comment: https://github.com/aquasecurity/trivy-java-db/pull/60/commits/b2d6433170875c82e599cd8c7abf3d1f154d192a